src/coap.c

```diff
@@ -522,4 +521,83 @@ NS_INTERNAL uint32_t coap_compose(struct ns_coap_message *cm,
   return 0;
 }
 
+/*
+ * FOSSA functions
+ */
+
+uint32_t ns_coap_send_message(struct ns_connection *nc,
```

Comment:
We have a documentation extraction from comments.

It works if a comment is right before a function declaration. 

I noticed that:

```
/* Options memory management functions */
void ns_coap_free_options(struct ns_coap_message *cm) {
```

Is meant to be a grouping comment but ended up being interpreted as a function comment.

Why not just add a short comment for all exported functions ?